OpenRA.Mods.Common/Projectiles/AreaBeam.cs

```diff
@@ -54,6 +54,9 @@ public class AreaBeamInfo : IProjectileInfo
 		[Desc("Can this projectile be blocked when hitting actors with an IBlocksProjectiles trait.")]
 		public readonly bool Blockable = false;
 
+		[Desc("Does the beam follow the target.")]
+		public readonly bool TracksTarget = false;
```

Comment:
`TrackTarget` (no `s`) would be more consistent with the other fields: `Render(s)Beam`, `Use(s)PlayerColor`.